test/e2e/editcrd/editcrd_test.go

```diff
@@ -67,7 +76,73 @@ var _ = Describe("test editcrd", Label("editcrd"), func() {
 					GinkgoWriter.Printf("failed to create v6 ippool %v with the same ip with another ippool %v\n", v6PoolName1, v6PoolName)
 				}
 			})
+
+		It(" failed to delete the ippool with existed ip and succeced delete the ippool with no existed ip",
```

Comment:
fail to delete the ippool with assigned ip and succeced delete the ippool with no-assigned ip